torchvision/csrc/cpu/video/Video.cpp

```diff
@@ -0,0 +1,393 @@
+
+#include "Video.h"
+#include <c10/util/Logging.h>
+#include <torch/script.h>
+#include "defs.h"
+#include "memory_buffer.h"
+#include "sync_decoder.h"
```

Comment:
nit: I know this is how `video_reader` is currently done, but I wonder if (for the future) it wouldn't be better to have this properly scoped with `decoder/defs.h` or something like that